mongo_connector/oplog_manager.py

```diff
@@ -668,16 +678,26 @@ def upsert_each(dm):
                             num_failed += 1
                         else:
                             raise
+                    if num % 10000 == 0:
+                        LOG.info("Upserted %d/%d docs from collection %s",
+                                 num + 1, total_docs, namespace)
```

Comment:
Good point. Since there's nothing we can do to make it more accurate how's this:
```python
LOG.info("Upserted %d out of approximately %d docs from collection %s",
         num + 1, total_docs, namespace)
...
LOG.info("Bulk upserting approximately %d docs from collection %s",
         total_docs, namespace)
```